### PR TITLE
update oauth proxy dev config base json

### PR DESCRIPTION
**Pull request by lukemarkwordtlibertyits**
# Description

Make the oauth-proxy's dev-config.base.json more generic.

# Related PRs

https://github.com/department-of-veterans-affairs/lighthouse-oauth-proxy-configs/pull/33

### CODE REVIEW

File: oauth-proxy/README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ This is a slim proxy for transforming and storing values from Okta's OpenID Conn
 
 ### Quick Start Local
 
+A functional dev-config.json file can be pulled from the oauth-proxy-configs repository.
```
Comment (lukemarkwordtlibertyits) [inline oauth-proxy/README.md:19]: Should I link the actual repository?

Comment (beaugrantham-va) [reply oauth-proxy/README.md:19]: Yes please add a link.

**Review by beaugrantham-va — CHANGES_REQUESTED** [review]
This wasn't included in the story, but please update the parent README section where it talks about dev-config.json.

https://github.com/department-of-veterans-affairs/vets-saml-proxy#running-locally-1

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ to have access to an okta authorization server and an api key for the server.
 
 To being you'll want to create a `dev-config.json` in the oauth-proxy subdirectory. That file should contain a 
 JSON object containing fields that corresponesd to the options document by the `--help` option. Once you've
-created that config you can run `npm start` to run the OAuth proxy with the code changes. 
+created that config you can run `npm start` to run the OAuth proxy with the code changes. A dev-config file can be found in the [oauth-proxy-configs](https://github.com/department-of-veterans-affairs/lighthouse-oauth-proxy-configs) repository, the developer will have to replace `okta_token` field with a functional token.
```
Comment (beaugrantham-va) [inline README.md:37]: This should replace the paragraph below, which references a repo not used by Lighthouse anymore (vets-contrib).

**Review by beaugrantham-va — APPROVED** [review]

**Pull request merged**